crcCards/use_case/TaskToEventAuto

```diff
@@ -1,11 +1,7 @@
-Class name: TaskToEventAuto
-Implements TaskToEvent
+Interface: TaskToEventAuto
 
 Responsibilities:
-    - Finds an available time in the Calendar using EventScheduler and confirms it with the user
-    - Converts the Task into an Event with the available time
+    - Finds an available time in the Calendar using EventScheduler
```

Comment:
So now it only check the time and is not responsible for creating an event? If so, the name of the interface might not be proper